lib/ogr/layer/extensions.rb

```diff
@@ -113,7 +113,7 @@ def geometry_from_extent
       #   from.
       # rubocop:todo Metrics/PerceivedComplexity
       def point_values(with_attributes = {}) # rubocop:todo Metrics/CyclomaticComplexity
-        return [[]] if feature_count.zero?
+        return [] if feature_count.zero?
```

Comment:
is this the actual bug fix?

Reply:
Yeah, the actual fix that's going to be used is in https://github.com/agrian-inc/ffi-gdal/pull/42/files, but on develop it's been moved over to this gem